### DISCUSSION TITLE
waste baskets show as AM at lower zooms

### DISCUSSION THREAD

**Discussion by tommycrock** [osmandapp/OsmAnd#11132] [Q&A] [answered]
For me, in walking mode, waste baskets show as "AM" on the map when more zoomed out. 
![Screenshot_20210313-084635_OsmAnd+](https://user-images.githubusercontent.com/5749884/111024837-ae935d80-83d8-11eb-9833-1d033d0b3197.jpg)

When I zoom in far enough they change to a trash can symbol. Is there a reason for it saying AM at some zooms? What does AM stand for? Thanks!

**Comment by tommycrock** [comment]
Well, I thought I had tried turning all of the details off, but today it's coming on and off with the OSM mapper assistant in Configure map>Details. Checking out a different area (Stockton) I also spotted OF on an office - more easily interpretable but not sure what it has to do with a mapper assistant. 
The location was https://osmand.net/go?lat=53.87054321309261&lon=-1.6934436250857061&z=20

**Comment by thompehw** [comment, marked as answer]
AM for "amenity"? The OSM tag is amenity=waste_basket. I think the mapper assistant option is making the feature more visible by showing AM when the zoom level is too low for the basket symbol to show.

**Reply by tommycrock** [reply]
Thanks. Good point.

I don't really understand why it would show me that rather than the trashcan symbol as it doesn't seem any smaller, and I don't understand why an OSM mapper would want to see it at zoom levels that a user wouldn't, but maybe I'm not understanding what the mapper assistant function is for

**Reply by tommycrock** [reply]
It seems these POIs weren't supported in osmand, or aren't in some map styles? "Stubs for unsupported (but present in maps) POIs. Displayed in OSMMap…"

**Reply by thompehw** [reply]
Sorry, tommycrock, I made your reply look random by deleting my own message from the board. The XML I quoted didn't seem to show.

By searching default.render.xml for "assistant" I suggested that these are the kinds of POI that the mapper assistant treats like "AM" and "OF":

<switch OSMMapperAssistant="true" minzoom="16" iconVisibleSize="7" intersectionSizeFactor="0.5">
	<case tag="amenity" value="" icon="stub_amenity" iconOrder="171"/>
	<case tag="club" value="" icon="stub_club" iconOrder="163"/>
	<case tag="craft" value="" icon="stub_craft" iconOrder="164"/>
	<case tag="emergency" value="" icon="stub_emergency" iconOrder="131"/>
	<case tag="healthcare" value="" icon="stub_healthcare" iconOrder="165"/>
	<case tag="historic" value="" icon="stub_historic" iconOrder="150"/>
	<case tag="office" value="" icon="stub_office" iconOrder="152"/>
	<case tag="man_made" value="" icon="stub_man_made" iconOrder="162"/>
	<case tag="shop" value="" icon="stub_shop" iconOrder="160"/>
	<case tag="tourism" value="" icon="stub_tourism" iconOrder="153"/>
</switch>

**Reply by thompehw** [reply]
OK, sorry, like this:

```
<switch OSMMapperAssistant="true" minzoom="16" iconVisibleSize="7" intersectionSizeFactor="0.5">
	<case tag="amenity" value="" icon="stub_amenity" iconOrder="171"/>
	<case tag="club" value="" icon="stub_club" iconOrder="163"/>
	<case tag="craft" value="" icon="stub_craft" iconOrder="164"/>
	<case tag="emergency" value="" icon="stub_emergency" iconOrder="131"/>
	<case tag="healthcare" value="" icon="stub_healthcare" iconOrder="165"/>
	<case tag="historic" value="" icon="stub_historic" iconOrder="150"/>
	<case tag="office" value="" icon="stub_office" iconOrder="152"/>
	<case tag="man_made" value="" icon="stub_man_made" iconOrder="162"/>
	<case tag="shop" value="" icon="stub_shop" iconOrder="160"/>
	<case tag="tourism" value="" icon="stub_tourism" iconOrder="153"/>
</switch>
```

For others new to Github, it has its own markup language https://guides.github.com/features/mastering-markdown/

I needed to surround the XML in triple single opening quotation marks... and use the preview tab available when composing on the Github website!
